src/core/currency/wallet/currency-wallet-callbacks.ts

```diff
@@ -417,6 +411,17 @@ export function watchCurrencyWallet(input: CurrencyWalletInput): void {
   checkChangesLoop()
 }
 
+// DEPRECATE: After all currency plugins implement new Confirmations API
+// This function is used to determine if the core should determine the confirmations
+// field value for a transaction.
```

Comment:
Please use the JavaDoc style:, with a summary of what the function does and an explicit deprecation marker:

```ts
/** 
 * Returns true if the core needs to calculate the transaction's confirmation state.
 * 
 * @deprecated Remove once all currency plugins support the new confirmations API.
 */
```

When I do a major breaking change in the core, I always so a search for the `@deprecated` tag, so this will be helpful in actually getting the code deleted.